web_app_drawer/static/src/css/web_app_drawer.css

```diff
@@ -9,70 +9,83 @@ body {
 /* Menu Bar */
 .css-main-nav {
     margin-bottom: 0 !important;
-    min-height: 28px !important;
+    min-height: 2em !important;
```

Comment:
Please avoid all `!important` rules, reserve those for custom themes.
http://stackoverflow.com/questions/3706819/what-are-the-implications-of-using-important-in-css#3706876